Library/Formula/hostdb.rb

```diff
@@ -1,14 +1,20 @@
-require 'formula'
-
 class Hostdb < Formula
   desc "Generate DNS zones and DHCP configuration from hostlist.txt"
-  homepage 'http://code.google.com/p/hostdb/'
-  url 'https://hostdb.googlecode.com/files/hostdb-1.004.tgz'
-  sha1 '65ec59c2c88b763813fa611d8fd28a45cd9d5278'
+  homepage "https://code.google.com/p/hostdb/"
+  url "https://hostdb.googlecode.com/files/hostdb-1.004.tgz"
+  sha256 "beea7cfcdc384eb40d0bc8b3ad2eb094ee81ca75e8eef7c07ea4a47e9f0da350"
 
   def install
-    bin.install Dir['bin/*']
-    doc.install Dir['docs/*']
-    (share+'examples'+name).install Dir['examples/*']
+    bin.install Dir["bin/*"]
+    doc.install Dir["docs/*"]
+    (share + "examples" + name).install Dir["examples/*"]
+  end
+
+  test do
+    system("#{bin}/mkzones -z " +
```

Comment:
Interpolate the strings and separate arguments e.g. `system "#{bin}/mkzones", "-z", "#{share}/examples/#{name}//example1/zoneconf.txt`